src/plugins/explore/public/application/app.tsx

```diff
@@ -106,10 +111,12 @@ export const ExploreApp: React.FC<{ setHeaderActionMenu?: (menuMount: any) => vo
     showSaveQuery: true,
   };
 
-  const showChart =
-    status === ResultStatus.READY ||
-    (status === ResultStatus.LOADING && !!rows?.length) ||
-    (status === ResultStatus.ERROR && !!rows?.length);
+  // Function to refresh data (for uninitialized state)
+  const onRefresh = () => {
+    if (services) {
+      dispatch(executeQueries({ services }) as any);
```

Comment:
when is services not defined? why `as any`?

Reply:
will fix the any type in the following PR together